### PR TITLE
[Client] Create chatting components

### PR DESCRIPTION
**Pull request by rian-yeji**

### CODE REVIEW

File: src/FRONTEND/meetview-client/src/components/chatting/ChattingList.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    {chatList}
+                </div>
+        )
+
+        return(
+            <Paper className={classes.chat_list}>
+                <InfiniteScroll
+                throttle={100}
+                threshold={300}
+                // dataLength={chatList.length} //This is important field to render the next data
```
Comment (dydtjr1128) [inline src/FRONTEND/meetview-client/src/components/chatting/ChattingList.js]: 주석

File: src/FRONTEND/meetview-client/src/components/chatting/ChattingTemplate.js [hunk 2]
```diff
@@ -1,19 +1,71 @@
...
-            </div>
-        );
+
+    onMessageReceive = (msg, topic) => {
+        console.log("Chatting Template - onMessageReceive");
+        //alert(JSON.stringify(msg) + " @ " + JSON.stringify(topic));
+        this.props.data.push(msg);
+        console.log(this.props.data);
+        // this.setState(prevState => ({
+        //     messages: [...prevState.messages, msg]
```
Comment (dydtjr1128) [inline src/FRONTEND/meetview-client/src/components/chatting/ChattingTemplate.js]: 주석

File: src/FRONTEND/meetview-client/src/components/chatting/ChattingList.js (hunk 1 above)
Comment (dydtjr1128) [inline src/FRONTEND/meetview-client/src/components/chatting/ChattingList.js]: 주석

File: src/FRONTEND/meetview-client/src/components/chatting/ChattingTemplate.js [hunk 3]
```diff
@@ -1,19 +1,71 @@
...
+        // }));
+    }
+
+    sendMessage = (newMsg) => {
+        console.log("Chatting Template - sendMessage");
```
Comment (dydtjr1128) [inline src/FRONTEND/meetview-client/src/components/chatting/ChattingTemplate.js]: Remove unnecessary log

File: src/FRONTEND/meetview-client/src/pages/Home.js [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
 import MainContainer from '../components/MainContainer';
+import ChattingTemplate from '../components/chatting/ChattingTemplate';
 
 export default function Home() {
 
     return(
         <div>
             <ListItems/>
             <MainContainer/>
+            {/* <ChattingTemplate /> */}
```
Comment (dydtjr1128) [inline src/FRONTEND/meetview-client/src/pages/Home.js]: Remove unnecessary remark

**Comment by jnsorn** [comment]
good

**Comment by haaaaaneul** [comment]
good!

**Pull request merged**